app/src/main/java/net/gsantner/markor/format/plaintext/PlaintextTextActions.java

```diff
@@ -102,10 +103,11 @@ public boolean onLongClick(View v) {
                     runRenumberOrderedListIfRequired(true);
                     return true;
                 }
-                default: {
-                    return runAction(_action, true, null);
+                case R.string.tmaid_common_insert_snippet: {
```

Comment:
Yes. `runAction` just delegates to `runCommonTextAction` which triggers the short press action. i.e. if we left the default case, long press in PlainText would trigger a short press if a long press action were not defined, instead of showing the tool tip as expected.